package.json

```diff
@@ -53,6 +52,7 @@
   },
   "devDependencies": {
     "@types/react-datepicker": "^3.0.2",
-    "@types/react-router-bootstrap": "^0.24.5"
+    "@types/react-router-bootstrap": "^0.24.5",
+    "react-scripts": "4.0.3"
```

Comment:
Do you think we should we look at upgrading to React 17 to go with the updated react-scripts?

Reply:
Discussed offline. This will be a different effort